src/index.html

```diff
@@ -22,6 +22,399 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="index.html"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link is-active"
+              href="index.html"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              data-qa="nav-hover"
+              href="index.html"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="index.html"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="catalog">
+      <div
+        class="card catalog__card"
+        data-qa="card"
```

Comment:
The `data-qa` attribute is used for test purposes. It's good practice to remove any test-specific attributes before deploying to production unless they are needed for end-to-end testing on the live site. [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules)